lib/builderator/interface/packer.rb

```diff
@@ -21,11 +21,21 @@ def initialize(*_)
 
         @packerfile ||= {
           :builders => [],
-          :provisioners => []
+          :provisioners => [],
+          'post-processors' => []
         }.tap do |json|
           Config.profile.current.packer.build.each do |_, build|
             build_hash = build.to_hash.tap do |b|
-              b[:tags] = Config.profile.current.tags
+              b[:tags] = Config.profile.current.tags unless Config.profile.current.tags.empty?
+            end
+
+            if build_hash[:type] == 'docker'
+              raise 'The Docker builder requires a base image' unless build_hash.key?(:image)
+
+              # The Docker builder requires one and only one of 'commit', 'discard', or 'export_path' set
+              if build_hash.keys.find_all { |k| [:commit, :discard, :export_path].include?(k) }.length != 1
```

Comment:
Good catch. I was looking at the difference between `#find_all` and `#select` as it applies to `Hash` even though `Hash.keys` returns an `Array`.

Fixed in efbdb6f.